uui-core/src/types/components/LabeledInput.tsx

```diff
@@ -21,4 +19,14 @@ export interface LabeledInputCoreProps
      * Can be used when component can't be wrapped by the LabeledInput, e.g. when form is layed out as table with labels and inputs placed into different columns
      */
     htmlFor?: string;
+    /** A value from LabeledInput children */
+    value?: any;
+    /** Maximum text length, in characters */
+    maxLength?: number;
+    /** Showing current text length, in characters and maxLength */
+    charCounter?: boolean
+    /** Additional text in the bottom of LabeledInput */
+    footNote?: ReactNode;
```

Comment:
let name it without camelCase -  `footnote`
And change comment to:    /** Additional info positioned at the bottom of LabeledInput */

Reply:
fixed